lib/spack/spack/schema/gitlab_ci.py

```diff
@@ -28,127 +30,104 @@
     ],
 }
 
+runner_attributes_schema_items = {
+    'image': image_schema,
+    'tags': {
+        'type': 'array',
+        'items': {'type': 'string'}
+    },
+    'variables': {
+        'type': 'object',
+        'patternProperties': {
+            r'[\w\d\-_\.]+': {
+                'type': 'string',
+            },
+        },
+    },
+}
+
+customizable_job_schema_items = {
```

Comment:
It looks like `runner_attributes_schema_items` and `customizable_job_schema_items` could be combined since they're always used together

Reply:
Thanks @zackgalbreath, that does appear to be the case.  I'll make that change.